tests/snappi_tests/variables.py

```diff
@@ -66,3 +69,153 @@
                      5: 0,
                      6: 1,
                      7: 0}
+
+
+def create_ip_list(value, count, mask=32, incr=0):
```

Comment:
Multiple uplink POs are getting same IP.

19:43:01 bgp_outbound_helper.duthost_bgp_config   L0461 INFO   | Creating PortChannel0 in str2-7250-lc1-2
19:43:01 bgp_outbound_helper.duthost_bgp_config   L0462 INFO   | Setting min_links to 2 for PortChannel0
19:43:01 bgp_outbound_helper.duthost_bgp_config   L0469 INFO   | Configuring IPs 20.3.1.0/31 , 2000:1:1:4::1/126 on PortChannel0 in str2-7250-lc1-2
19:43:01 bgp_outbound_helper.duthost_bgp_config   L0460 INFO   |
 
19:43:01 bgp_outbound_helper.duthost_bgp_config   L0461 INFO   | Creating PortChannel1 in str2-7250-lc1-2
19:43:01 bgp_outbound_helper.duthost_bgp_config   L0462 INFO   | Setting min_links to 2 for PortChannel1
19:43:01 bgp_outbound_helper.duthost_bgp_config   L0469 INFO   | Configuring IPs 20.3.1.0/31 , 2000:1:1:4::1/126 on PortChannel1 in str2-7250-lc1-2

Reply:
implemented